packages/design-system/src/components/Tooltip/Tooltip.test.jsx

```diff
@@ -24,11 +24,6 @@ describe('Tooltip', function () {
     expect(tooltip.wrapper).toMatchSnapshot();
   });
 
-  it('renders inverse tooltip', () => {
-    const tooltip = render({ inversed: true });
-    expect(tooltip.wrapper).toMatchSnapshot();
-  });
-
```

Comment:
Add this test back as we're having `inversed` tooltip flag for the different tooltip color